services/apis/fence/fenceTasks.js

```diff
@@ -333,9 +333,6 @@ module.exports = {
    * @returns {Promise<string>} - std out from the fence-create script
    */
   async forceLinkGoogleAcct(userAcct, googleEmail) {
-    // hit link endpoint to ensure a proxy group is created for user
-    await I.sendGetRequest(fenceProps.endpoints.linkGoogle, userAcct.accessTokenHeader);
-
     // run fence-create command to circumvent google and add user link to fence
     const cmd = `fence-create force-link-google --username ${userAcct.username} --google-email ${googleEmail}`;
```

Comment:
we dont need this fence-creatte command anymore

Reply:
I have already removed those line